packages/vuetify/src/directives/ripple/index.ts

```diff
@@ -283,6 +287,9 @@ function updateRipple (el: HTMLElement, binding: RippleDirectiveBinding, wasEnab
     el._ripple.class = value.class
   }
 
+  const allowedKeys = (typeof value === 'object' ? value.keys : null) ?? [keyCodes.enter, keyCodes.space]
```

Comment:
We're already calling **isObject(value)** above, can we just assign that to a const and reuse it as opposed to calling typeof value === 'object'?